web/less/base.less

```diff
@@ -294,7 +300,7 @@ textarea
             color: @greytext;
             text-decoration: none;
         }
-        a:visisted
+        a:visited
```

Comment:
poor use of less. The regular, hover, and visited styles are all the same. Could just be a single block with all 3 selectors. meh

Reply:
this wont cause a regression because it doesnt actually change the styling